shell/common/engine.h

```diff
@@ -905,6 +905,8 @@ class Engine final : public RuntimeDelegate, PointerDataDispatcher::Delegate {
     return runtime_controller_.get();
   }
 
+  const std::shared_ptr<VsyncWaiter> GetVsyncWaiter() const;
```

Comment:
given that engine will outlive the vsync waiter, can we hand a reference rather than handing out a shared pointer.